x-pack/test/api_integration/deployment_agnostic/apis/observability/apm/time_range_metadata/time_range_metadata.spec.ts

```diff
@@ -55,37 +54,37 @@ export default function ApiTest({ getService }: FtrProviderContext) {
     };
   }
 
-  registry.when('Time range metadata without data', { config: 'basic', archives: [] }, () => {
-    it('handles empty state', async () => {
-      const response = await getTimeRangeMedata({
-        start,
-        end,
-      });
+  describe('Time range metadata', () => {
+    describe('without data', () => {
+      it('handles empty state', async () => {
+        const response = await getTimeRangeMedata({
+          start,
+          end,
+        });
 
-      expect(response.isUsingServiceDestinationMetrics).to.eql(false);
-      expect(response.sources.filter((source) => source.hasDocs)).to.eql([
-        {
-          documentType: ApmDocumentType.TransactionEvent,
-          rollupInterval: RollupInterval.None,
-          hasDocs: true,
-          hasDurationSummaryField: false,
-        },
-      ]);
+        expect(response.isUsingServiceDestinationMetrics).to.eql(false);
+        expect(response.sources.filter((source) => source.hasDocs)).to.eql([
+          {
+            documentType: ApmDocumentType.TransactionEvent,
+            rollupInterval: RollupInterval.None,
+            hasDocs: true,
+            hasDurationSummaryField: false,
+          },
+        ]);
+      });
     });
-  });
 
-  registry.when(
-    'Time range metadata when generating data with multiple APM server versions',
-    { config: 'basic', archives: [] },
-    () => {
+    describe('when generating data with multiple APM server versions', () => {
       describe('data loaded with and without summary field', () => {
+        let apmSynthtraceEsClient: ApmSynthtraceEsClient;
```

Comment:
We can instantiate `apmSynthtraceEsClient` at the top `describe` to avoid doing it two times

Reply:
good catch